packages/extension-chakra-storefront/src/components/product-scroller/index.jsx

```diff
@@ -162,11 +164,13 @@ const ProductScroller = forwardRef(
                                     id: 'product_scroller.assistive_msg.scroll_right',
                                     defaultMessage: 'Scroll products right'
                                 })}
-                                icon={<ChevronRightIcon color="black" />}
                                 borderRadius="full"
-                                colorScheme="whiteAlpha"
+                                bg="white/36"
+                                _hover={{bg: 'white/48'}}
                                 onClick={() => scroll(1)}
-                            />
```

Comment:
There are some components that are using `sx` prop in this component. Would you mind replacing it with `css` prop for consistency?